install/logging.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Renaming this file to storageService.logging.conf would make sense?

Reply:
Done.